java/src/jmri/jmrit/beantable/LogixTableAction.java

```diff
@@ -428,6 +485,44 @@ ActionListener init(BeanTableFrame f) {
         }.init(f));
         menu.add(item);
         menuBar.add(menu, pos + offset + 1); // add this menu to the right of the previous
+        getSelectionMode();
+    }
+
+    /**
+     * Get the saved mode selection, if any, and click the appropriate menu item.
+     * <p>
+     * If the mode is not valid, use the traditional tabbed drag and drop selection table format by clicking on the menuMulti button.
+     * The doClick results in a call to setSelectionMode() which updates the _selectionMode variable.
+     */
+    void getSelectionMode() {
```

Comment:
Yes, although I would call setSelectionMode(mode) instead of calling the doClick() methods of the related menu items, and have setSelectionMode(mode) fire a property changed event. The menu items can listen for that event and set themselves appropriately (this has a design advantage in that the LogixTableAction class does not need to know about the menu items except in the one method that creates them).